pkg/gormutil/gormutil.go

```diff
@@ -0,0 +1,33 @@
+package gormutil
+
+import (
+	"sync"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var (
+	once      sync.Once
+	singleton *gorm.DB
+)
+
+// OpenDBConnection opens a DB connection.
+func OpenDBConnection(connStr string, config gorm.Config) (*gorm.DB, error) {
```

Comment:
should we add mysql to the method name ? what do you think ?

Reply:
You're right!